ovirt/resource_ovirt_vm_template.go

```diff
@@ -301,6 +311,15 @@ func resourceOvirtTemplateCreate(d *schema.ResourceData, meta interface{}) error
 		builder.Memory(int64(memory.(int)) * int64(math.Pow(2, 20)))
 	}
 
+	if guaranteed_memory, ok := d.GetOk("guaranteed_memory"); ok {
+		//set guaranteed memory
+		builder.MemoryPolicy(
+			ovirtsdk4.NewMemoryPolicyBuilder().Guaranteed(int64(guaranteed_memory.(int)) * int64(math.Pow(2, 20))).
```

Comment:
shouldn't "int64(math.Pow(2, 20)"  be a constant? I mean we are using it all around, it will be much clearer

Reply:
looks like we are using it every time we want to convert MB to Byte (what is SDK requires), we can change it to bytesInMega constant.

Reply:
done